File: HanYeojin/Assignment2/yeojin/cafe-server/src/main/java/com/yeojin/cafe/controller/MenuController.java
package com.yeojin.cafe.controller;

import com.yeojin.cafe.dto.MenuResponseDto;
import com.yeojin.cafe.service.MenuService;
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;
import lombok.RequiredArgsConstructor;
import java.util.List;

@RestController
@RequestMapping("/api/menus")
@RequiredArgsConstructor
public class MenuController {
    private final MenuService menuService;

    // 전체 메뉴 조회 - GET /api/menus
    @GetMapping
    public List<MenuResponseDto> getAllMenus() {
        return menuService.getAllMenus();
    }
}

File: assignment2/MinSeungho/cafe/src/main/kotlin/com/seungho/cafe/controller/MenuController.kt
package com.seungho.cafe.controller

import com.seungho.cafe.dto.MenuResponse
import com.seungho.cafe.service.MenuService
import org.springframework.http.ResponseEntity
import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RequestMapping
import org.springframework.web.bind.annotation.RestController

@RestController
@RequestMapping("/api/menus")
class MenuController (
    private val menuService: MenuService
){
    @GetMapping
    fun getAllMenus() : ResponseEntity<List<MenuResponse>> {
        val menus = menuService.getAllMenus()
        return ResponseEntity.ok(menus)
    }
}